modules/web-console/frontend/app/configuration/store/effects.js

```diff
@@ -210,7 +210,7 @@ export default class ConfigEffects {
 
         this.errorNotificationsEffect$ = this.ConfigureState.actions$.pipe(
             filter((a) => a.error),
-            tap((action) => this.IgniteMessages.showError(action.error)),
+            tap((action) => this.IgniteMessages.showError(action.error.title, action.error.message)),
```

Comment:
I don't remember title field in error object. Is this correct?

Reply:
Fixed